DataHub service-level agreement (#2663)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-undefined-Service-Terms.md

---
date: 2024-12-05
title: Further improvements to Service Terms
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
product_area: Platform services
component:
  - value: component-docs-123
    label: Documentation
build_artifact:
---
The section on [Service terms](/service-terms/) has been further enhanced. It now also contains

* the [Support service-level agreement](/service-terms/service-level/#support-sla),
* the [DataHub service-level agreement](/service-terms/service-level/#datahub-sla),

> * the [Compatibility policy](/service-terms/compatibility-policy/) (moved from "Getting started"),

Reviewer: ```suggestion
* the [Compatibility policy](/service-terms/compatibility-policy/) (moved from "Technical concepts"),
```

---
date: 2024-12-05
title: Further improvements to Service Terms
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
product_area: Platform services
component:
  - value: component-docs-123
    label: Documentation
build_artifact:

>   - value: tc-QHwMfWtBk7

Reviewer: ```suggestion
```

---
date: 2024-12-05
title: Further improvements to Service Terms
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
product_area: Platform services
component:
  - value: component-docs-123
    label: Documentation
build_artifact:
  - value: tc-QHwMfWtBk7

>     label: cumulocity

Reviewer: ```suggestion
```

---
date: 2024-12-05
title: Further improvements to Service Terms
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
product_area: Platform services
component:
  - value: component-docs-123
    label: Documentation
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity

>     label: cdh

Reviewer: ```suggestion
```

---
date: 2024-12-05
title: Further improvements to Service Terms
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
product_area: Platform services
component:
  - value: component-docs-123
    label: Documentation
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity

>   - value: tc-H-tuq-8Es

Reviewer: ```suggestion
```

---
date: 2024-12-05
title: Further improvements to Service Terms
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
product_area: Platform services
component:
  - value: component-docs-123
    label: Documentation

>     label: DataHub

Reviewer: ```suggestion
```

---
date: 2024-12-05
title: Further improvements to Service Terms
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
product_area: Platform services
component:

>   - value: component-9vjGQz8Ag

Reviewer: ```suggestion
  - value: component-docs-123
```

>     label: Core platform

Reviewer: ```suggestion
    label: Documentation
```

>   - value: component-A8vMaVaTg

Reviewer: ```suggestion
```
